mandatory/1-fix-functions.js

```diff
@@ -31,20 +31,20 @@ function greaterThan10(num) {
 }
 
 function sortArray(letters) {
-  let sortedLetters = letters;
+  let sortedLetters = letters.sort;
 
   return sortedLetters;
 }
 
 function first5(numbers) {
-  let sliced;
+  let sliced = numbers.sliced(0.5);
 
   return sliced;
 }
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element = arr[index];
 
```

Comment:
I think you can reduce this function to one line if we know that we want the third index everytime.